start/data/stashcache.md

```diff
@@ -62,29 +96,79 @@ transfer this file into your current working directory on the compute host would
 
 	stashcp /osgconnect/public/username/samples/sample01.dat  ./
 
-# Use StashCache To Tranfer Large Output Files 
+# Use StashCache To Tranfer Larger Output Files To `/public`
 
-To transfer job output to StashCache after your analysis:
+To transfer larger output files (>1GB) back to your `/public` directory (which 
+is necessary to later access your results):
 
-1) Ensure that your job's submit script indicates the necessary requirements to 
-make `stashcp` available by including the following lines:
+1) Add the necessary details to your HTCondor submit file to tell 
+HTCondor that your jobs must run on executes nodes that 
+have access StashCache and to OSG Connect modules.
 
+		# StashCache submit file example
+		
+		log = my_job.$(Cluster).$(Process).log
+		error = my_job.$(Cluster).$(Process).err
+		output = my_job.$(Cluster).$(Process).out
+		
+		# ensure jobs have access to StashCache
 		+WantsStashCache = true
-		requirements = OSGVO_OS_STRING == "RHEL 7" && Arch == "X86_64" && HAS_MODULES == True
+		requirements = (OSGVO_OS_STRING =?= "RHEL 7") && (HAS_MODULES =?= True)
+		
+		...other submit file details...
+
+2) Use `stashcp` command to transfer the data files back to `/public`. You will 
+need to prepend your `/public` directory path with `stash://osgconnect` as follows:
+
+		#!/bin/bash
+	
+		# commands to be executed in job   
+		
+		...   
+		
+		# load module   
+		module load stashcache   
+		
+		# transfer large output to public
+		stashcp file_name stash:///osgconnect/public/username/path/file_name
 
-2) Use `stashcp` to transfer the data files back to StashCache. You will 
-need to prepend your stash location with `stash://` as follows:
+	For example, if you wish to transfer `output.dat` to the directory 
+	`/public/username/output/` then the `stashcp` command would be:
 
-		module load stashcache
-		stashcp <file_name> stash:///osgconnect/public/username/path/<file_name>
+		stashcp output.dat stash:///osgconnect/public/username/output/output.dat
 
-For example, if you wish to transfer `output.dat` to the directory 
-`/public/username/output/` then the `stashcp` command would be:
+	**Notice that the output file name `output.dat` must be included at the end of the 
+	`/public` path where the file will be transferred.**
```

Comment:
does the full path up to the filename need to exist already? i.e. people couldn't do: 
```
unique=`date +%s`
staschp out.dat stash:///osgconnect/public/username/$unique/out.dat
```
?

Reply:
I don't know, but my guess is that the path would need to exist first, like a standard `cp` command, but i can test this

Reply:
Christina did some tests, it looks like the path does not need to exists before hand so yes, users could instead use the example that Christina provided, but I think it might be easier on users to prepend the file name that way they can do some thing like `ls -lh *.out.dat` or for ease of transferring output off the system

Reply:
@CarrieBrown what are your thoughts?